jib-cli/README.md

```diff
@@ -120,7 +121,23 @@ This command follows the following pattern:
 ```
 jib jar --target <image name> path/to/myapp.jar [options]
 ```
+### Quickstart
+1. Have your JAR (thin or fat) ready. We will be using the [Spring Petclinic](https://projects.spring.io/spring-petclinic/) JAR in this Quickstart.
+   ```
+    $ git clone https://github.com/spring-projects/spring-petclinic.git
+    $ cd spring-petclinic
+    $ ./mvnw package
+   ```
+2. Containerize your JAR using the `jar` command. In the default mode (exploded), the entrypoint will be set to `java -cp /app/dependencies/:/app/explodedJar/ HelloWorld`
+   ```
+    $ jib jar --target=docker://cli-jar-quickstart target/spring-petclinic-2.4.2.jar
```

Comment:
Another way is
```suggestion
    $ jib jar --target=docker://cli-jar-quickstart target/spring-petclinic-*.jar
```
only if it expands only to a single file. If so, it can reduce the friction.

Reply:
Oh nice. This works. I'll make the change.